### PR TITLE
Updated Release notes and host version

### PR DESCRIPTION
**Pull request by v-smanchem**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

**Comment by VpOfEngineering** [comment]
> ### Issue describing the changes in this PR
> resolves #issue_for_this_pr
> 
> ### Pull request checklist
> * [ ]  My changes **do not** require documentation changes
>   
>   * [ ]  Otherwise: Documentation issue linked to PR
> * [ ]  My changes **should not** be added to the release notes for the next release
>   
>   * [ ]  Otherwise: I've added my notes to `release_notes.md`
> * [ ]  My changes **do not** need to be backported to a previous version
>   
>   * [ ]  Otherwise: Backport tracked by issue/PR #issue_or_pr
> * [ ]  My changes **do not** require diagnostic events changes
>   
>   * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
> * [ ]  I have added all required tests (Unit tests, E2E tests)
> 
> ### Additional information
> Additional PR information

Also - in the future - please fill this section out and set the description. Let me know if any part is unclear

File: release_notes.md [hunk 1]
```diff
@@ -3,9 +3,7 @@
 - My change description (#PR)
 -->
 - Update Python Worker Version to [4.8.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.8.0)
-- Update Java Worker Version to [2.7.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.7.0)
-- Update Node.js Worker Version to [3.5.1](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.5.1)
 - Update ApplicationInsights packages to 2.21.0 (#8838)
```
Comment (liliankasem) [inline release_notes.md:6]: Just wondering why all the notes aren't cleared?

**Comment by surgupta-msft** [comment]
Bit confused about this PR. The Host already has a commit merged on 11/14 updating the minor version to 15 - https://github.com/Azure/azure-functions-host/blob/83081e19b68768f5cb8da360c3b4f6e879e49e67/build/common.props#L7

Is this PR still needed or should we close/abandon it?

**Comment by liliankasem** [comment]
> Bit confused about this PR. The Host already has a commit merged on 11/14 updating the minor version to 15 -
> 
> https://github.com/Azure/azure-functions-host/blob/83081e19b68768f5cb8da360c3b4f6e879e49e67/build/common.props#L7
> 
> Is this PR still needed or should we close/abandon it?

@v-smanchem  bump

**Pull request closed without merging**